public/app/features/dashboard/components/DeleteDashboard/DeleteDashboardModal.tsx

```diff
@@ -29,7 +29,7 @@ const DeleteDashboardModalUnconnected = ({ hideModal, cleanUpDashboardAndVariabl
 
   const [, onConfirm] = useAsyncFn(async () => {
     await deleteDashboard(dashboard.uid, true);
-    cleanUpDashboardAndVariables();
+    // cleanUpDashboardAndVariables();
```

Comment:
Should this line be commented?

Reply:
This method does [a lot of actions](https://github.com/grafana/grafana/blob/main/public/app/features/dashboard/state/actions.ts#L32-L48) to clean up the dashboard including removing panels and variables and such. If it runs, recovering it would be difficult

Reply:
I would expect all of those deletions to be related to the frontend. The entire business logic should be in the backend endpoint. @juanicabanas can you help us confirming what is the purpose of that cleanup method?

Reply:
let me take a look